app/controllers/events/view/tickets/add-order.js

```diff
@@ -52,6 +52,47 @@ export default Controller.extend({
           order.tickets.removeObject(ticket);
         }
       }
+    },
+    async placeOrder() {
+      let order = this.get('model.order');
+      let event = order.get('event');
+      order.tickets.forEach(ticket => {
+        let numberOfAttendees = ticket.orderQuantity;
+        while (numberOfAttendees--) {
+          this.get('model.attendees').addObject(this.store.createRecord('attendee', {
+            firstname : 'John',
+            lastname  : 'Doe',
+            email     : 'johndoe@example.com',
+            event,
+            ticket
+          }));
+        }
+      });
+      try {
+        this.set('isLoading', true);
```

Comment:
I think you should move this line to the first line of this method i.e. link 57.